### ISSUE TITLE
Cannot build solution from fresh repo clone

### ISSUE THREAD

**Issue by jbennink** [Azure-Samples/AzureMapsCodeSamples#47]
### This issue is for a: (mark with an `x`)
```
- [X] bug report
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
Clone repo, open solution, press F5 to build. Build error:

Severity	Code	Description	Project	File	Line	Suppression State
Error		Build:File 'C:/data/git/AzureMapsCodeSamples/AzureMapsCodeSamples/Common/typings/azure-maps-control.d.ts' not found.	AzureMapsCodeSamples	C:\data\git\AzureMapsCodeSamples\AzureMapsCodeSamples\Experimental\Solar Terminator Module\SolarTerminatorModule.ts	25	

This file is not part of the repo, but is mentioned in the csproj.

### Expected/desired behavior
Build out of the box 

### OS, Browser and Version?
> Windows 10 (2004)
> Edge

**Comment by rbrundritt** [comment]
This repo is not meant to be cloned and ran, but instead the individual code samples should be ran on their own. 

That said, I'm in the process of breaking out the typescript samples/modules into their own standalone projects. Once that is done, that will likely address the issue you are seeing. 

You could also delete the solar terminator module as it is an experimental item that isn't fully functional yet.
